chia/full_node/block_store.py

```diff
@@ -488,3 +488,23 @@ async def get_random_not_compactified(self, number: int) -> List[int]:
         heights = [int(row[0]) for row in rows]
 
         return heights
+
+    async def count_compactified_blocks(self) -> int:
```

Comment:
Is there a benchmark for this? Looks pretty slow. Does it work on both v1 and v2 schemas? This needs a test for sure

Reply:
`"is_fully_compactified tinyint,"` exists on both schema versions
Benchmark data is in another comment on the PR (also, moved to a separate RPC method now that will alleviate any concerns about it running too much, but its a few ms)